### DISCUSSION TITLE
SFSC - SFCC Connector

### DISCUSSION THREAD

**Discussion by shayunBBD** [SalesforceCommerceCloud/sfcc-ci#405] [Team Posts]
Hello,

I succeed to upload the int_service_cloud to my sb and I can see it in the cartridges ist of my site but I still don't see the Custom-SCC-SyncCustomers step for my created job.

Please help me
Thank you

**Comment by zlatinz** [comment]
See https://documentation.b2c.commercecloud.salesforce.com/DOC1/topic/com.demandware.dochelp/DSS/UpdatingTheCartridgePath.html and https://github.com/SalesforceCommerceCloud/service-cloud-connector/blob/release/docs/setupB2CCommerceJobs.md#install-the-int_service_cloud-cartridge

**Comment by tavaresqueiroz** [comment]
Take a specifically look at:
"Toggling the code-versions force Salesforce B2C Commerce to scan the cartridge path for the site and parse any job-steps configured within the cartridges identified in the cartridge-path."
In the links sent by @zlatinz
